runsc/boot/filter/config/config.go

```diff
@@ -41,6 +42,7 @@ type Options struct {
 	NVProxy               bool
 	TPUProxy              bool
 	ControllerFD          uint32
+	PluginNetwork         bool
```

Comment:
If this modifies the seccomp rules, and it sounds like it does, please add it to the `ConfigKey` function below.

Reply:
Thanks for reminding. We have updated corresponding configKey and seccomp warnings as well.